modules/tinymce/src/core/main/ts/delete/BlockMergeBoundary.ts

```diff
@@ -1,6 +1,7 @@
 import { Optional, Optionals } from '@ephox/katamari';
 import { Compare, PredicateFind, SugarElement, SugarNode } from '@ephox/sugar';
 
+import Schema from '../api/html/Schema';
```

Comment:
off-topic, but I wonder if we can start using `import type` 🤔

Reply:
I'd like to not import run time code if I don't need it.

Reply:
I remember I tried to use that and Lee told me that since it was a modern TS feature older TS versions wouldn't be able to consume our types. Not sure if that is true or not haven't tested to import type with `type` and compare the .d.ts files to see if anything changed since that is the only thing that could be breaking things for third party.